examples/react/trimmed/src/useMosaicReactTable.tsx

```diff
@@ -1,40 +1,127 @@
-// This is an example of the React-specific hook that'd be used to allow
```

Comment:
These changes should not be happening. You are trying to circumvent React here in the efforts of better rendering perf.

Let's incrementally make these changes only if it becomes a performance bottleneck.